plugins/discovery-file/build.gradle

```diff
@@ -27,3 +31,36 @@ bundlePlugin {
         into 'config'
     }
 }
+
+task seedTask << { }
```

Comment:
I don't think you need this extra task. Just pass your setupSeedNode task to ClusterFormationTasks